src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -152,16 +148,16 @@ public MainMenu() {
 		}
 
 		if (singleUser) {
-			LOGGER.info("Logging: Single User mode.");
-			myUser = new User(ADMIN_STR, new UserGroup(ADMIN_STR, ""), ADMIN_STR, "");
+			logger.info("Logging: Single User mode.");
+			myUser = new User("admin", new UserGroup("admin", ""), "admin", "");
 			MDC.put("OHUser", myUser.getUserName());
 			MDC.put("OHUserGroup", myUser.getUserGroupName().getCode());
 		} else {
 			// get an user
-			LOGGER.info("Logging: Multi User mode.");
+			logger.info("Logging: Multi User mode.");
```

Comment:
correct form is with LOGGER

Reply:
I restored the current informatici version